src/Interfaces.hs

```diff
@@ -95,5 +96,5 @@ retroactivelyRegisterInInterface ctx interface =
   either (\e -> error e) id resultCtx
   where
     env = contextGlobalEnv ctx
-    impls = recursiveLookupAll (getPath (binderXObj interface)) lookupImplementations env
+    impls = lookupMany Everywhere lookupImplementations (getPath (binderXObj interface)) env
```

Comment:
Love the fact that the more specific arguments (binder, env) comes later--this makes it possible to use partial application effectively in higher order functions.